design/ux/session_keys_on_ten_gateway.md

```diff
@@ -20,48 +20,45 @@ The TG already manages VKs on behalf of users, so adding SKs is relatively strai
 
 The advantage of the TG managing SKs is that it can return the funds to the EOA anytime.
 
-
 Another advantage is that they can be reused between sessions to avoid unnecessary transactions.
 
 
 ### The flow
 
-The logic of the game UI (javascript) will call an endpoint on the TG that will return a value transfer transaction that the user must sign in their wallet, together with the address of the SK.
+#### Create
+The logic of the game UI (javascript) will call an `/session_key/create` endpoint on the TG, which will return the account address corresponding to the SK. 
+The game UI will create a value transfer transaction from the main account to this address with enough money to prepay for the moves. The user must sign it with their wallet.
+
+The game will call `/session_key/activate/${sk_account}`
+
+After the initialisation, the game will create unsigned move transactions and submit them to the TG. 
```

Comment:
Maybe worth saying explicitly how that tx part works, I guess it's like /session_key/submit_tx or something? And the payload is similar to eth_sendRawTransaction.

Reply:
in my mind it would be a normal transaction, sent via the normal rpc. Only that the signature fields are 0 or something

Reply:
Ah ok fair, yeah if the wallets let you do that then that'd work nicely